CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -1,3 +1,11 @@
+# v1.9.3 - Changelog since v1.9.2
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix missing libedit.so.2 error ([#1177](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1177), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
There is also the Update go version to 1.20.3 for k/k 1.27: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1181

Reply:
Added